sugoi-api-core/src/main/java/fr/insee/sugoi/core/model/PasswordChangeRequest.java

```diff
@@ -0,0 +1,58 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package fr.insee.sugoi.core.model;
+
+import java.util.Map;
+
+public class PasswordChangeRequest {
+
```

Comment:
We may need other properties/metadata in this request. Adding a map<String, String> of preperties could be useful